Firestore/core/src/firebase/firestore/local/local_serializer.cc

```diff
@@ -254,15 +255,15 @@ QueryData LocalSerializer::DecodeQueryData(
   SnapshotVersion version =
       rpc_serializer_.DecodeVersion(reader, proto.snapshot_version);
   ByteString resume_token(proto.resume_token);
-  Query query = InvalidQuery();
+  Target target = InvalidTarget();
```

Comment:
All of our other types in here are just default constructible. This `InvalidTarget` thing doesn't seem that useful. Let's remove it and add a default constructor to `Target`.